modules/core/src/main/java/org/apache/synapse/mediators/builtin/LogMediator.java

```diff
@@ -244,13 +244,9 @@ public String getSeparator() {
     }
 
     public void setSeparator(String separator) {
-        if (separator.equals("\\n")) {
-            this.separator = "\n";
-        } else {
-            this.separator = separator;
-        }
+        this.separator = separator.replace("\\n", "\n").replace("\\t", "\t");
```

Comment:
What about other characters other than \n and \t? Is there any other possible characters? If there are, and next time when a user use such, we need to update the code again.

Reply:
We cannot handle this in front end.